src/bhv_basic_move.cpp

```diff
@@ -108,10 +117,17 @@ Bhv_BasicMove::execute( PlayerAgent * agent )
     if ( ! Body_GoToPoint( target_point, dist_thr, dash_power
                            ).execute( agent ) )
     {
-        Body_TurnToBall().execute( agent );
+      if (! Body_TurnToBall().execute( agent )) {
+	success = false;
+      } else {
+	success = maybe_success;
```

Comment:
The logical flow of this method is quite hard to interpret. Is there a way to re-organize it?